lib/rack_middleware/application.rb

```diff
@@ -0,0 +1,9 @@
+module RackMiddleware
+  class Application
+    # rubocop:disable Lint/UnusedMethodArgument
+    def call(env)
```

Comment:
You can replace env with _env and rubocop stops complaining

Reply:
I thought will be better to keep param name 'couse it is standard, fixed